### PR TITLE
add describe network interfaces

### PR DESCRIPTION
**Pull request by sfc-gh-dwyleczukstern**

### CODE REVIEW

File: src/connectors/aws_collect.py [hunk 1]
```diff
@@ -175,6 +175,33 @@
...
+        ('ipv6_addresses', 'VARIANT'),
+        ('mac_address', 'STRING'),
+        ('network_interface_id', 'STRING'),
+        ('owner_id', 'STRING'),
+        ('private_ip_address', 'STRING'),
+        ('private_dns_name', 'STRING')
+        ('private_ip_addresses', 'VARIANT'),
+        ('requester_id', 'STRING'),
+        ('requester_managed', 'BOOLEAN'),
+        ('source_dest_check', 'STRING'),
```
Comment (sfc-gh-afedorov) [inline src/connectors/aws_collect.py:199]: docs say
> SourceDestCheck -> (boolean)

File: src/connectors/aws_collect.py [hunk 2]
```diff
@@ -638,6 +665,34 @@
...
+                    'Ipv6Addresses': 'ipv6_addresses',
+                    'RequesterId': 'requester_id',
+                    'Attachment': 'attachment',
+                    'Association': 'attachment',
+                    'AvailabilityZone': 'availability_zone',
+                    'Description': 'description',
+                    'MacAddress': 'mac_address',
+                    'NetworkInterfaceId': 'network_interface_id',
+                    'OwnerId': 'owner_id',
+                    'PrivateIpAddress': 'private_ip_address',
```
Comment (sfc-gh-afedorov) [inline src/connectors/aws_collect.py:683]: missing `OutpostArn`

**Review by sfc-gh-afedorov — APPROVED** [review]
lgtm, should be good to merge with the two changes to the collector and please also add the CREATE TABLE to migration [here](https://github.com/snowflakedb/SnowAlert/blob/v1.10.0/migrations/v1_9_6-v1_10_0.md)

**Comment by sfc-gh-dwyleczukstern** [comment]
Updated based on requested feedback

**Pull request merged**